sounds/views.py

```diff
@@ -188,6 +188,7 @@ def front_page(request):
 
     rss_cache = cache.get("rss_cache_bw" if using_beastwhoosh(request) else "rss_cache", None)
     trending_sound_ids = cache.get("trending_sound_ids", None)
+    trending_pack_ids = cache.get("trending_pack_ids", None)
```

Comment:
If cache is not set this will become `None`. in the template we check if these lists have elements and only display the corresponding sections if that is the case. I think this is clear enough for me but we could refactor that if you have a suggestion to make it better.

Reply:
right - my question was asking what do we do if it's empty? Are we happy to show nothing on the front page if these caches aren't set, or do we want to set them if needed?

Reply:
Oh, I'd say we're happy. These caches are set in a cronjob so will be available unless something very bad happens :)

Reply:
but this happens every time we restart webservers - we lose the blog caches. I thought that since we were working on this part of the site we could fix this problem

Reply:
hmm, are you sure about that? I think this does not happen as the caches are still in memcached, aren't they?